checker/examples/units-extension/Makefile

```diff
@@ -10,5 +10,5 @@ compile: $(FILES)
 	$(JAVAC) $(FILES)
```

Comment:
It would make sense to add either this example or a variant of it to the normal test suite - to ensure we don't miss issues in extensions again.